impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -37,6 +37,10 @@ void RuntimeEffectContents::SetTextureInputs(
   texture_inputs_ = std::move(texture_inputs);
 }
 
+bool RuntimeEffectContents::CanAcceptOpacity(const Entity& entity) const {
+  return false;
```

Comment:
Should this just default to returning false on the base class?

Reply:
It does, but not on color source. I'm somewhat indifferent, we're probably not going to add any more content types.

Reply:
Contents (false) -> ColorSource (true) -> RuntimeEffectContents(false).

Its sort of an odd hierarchy for sure